src/main/java/org/mule/extension/http/internal/listener/HttpResponseFactory.java

```diff
@@ -171,8 +164,10 @@ private void addUserHeaders(HttpListenerResponseBuilder listenerResponseBuilder,
 
     headers.keySet().forEach(key -> {
       if (!supportsTransferEncoding && HEADER_TRANSFER_ENCODING.equalsIgnoreCase(key)) {
-        logger.debug(
-                     "Client HTTP version is lower than 1.1 so the unsupported 'Transfer-Encoding' header has been removed and 'Content-Length' will be sent instead.");
+        if (logger.isDebugEnabled()) {
+          logger.debug(
+                       "Client HTTP version is lower than 1.1 so the unsupported 'Transfer-Encoding' header has been removed and 'Content-Length' will be sent instead.");
```

Comment:
this is just a static constant string. There is no gain wrapping this in a `if (logger.isDebugEnabled())`